documentation/src/docs/metrics.md

```diff
@@ -7,491 +7,466 @@ You may obtain a copy of the License at
+
+   ```
+    pravega_segmentstore.segment.create_latency_ms
+    pravega_segmentstore.segment.read_latency_ms
+    pravega_segmentstore.segment.write_latency_ms
+
+  ```
+
+- Segment Store global and per-segment Read/Write Metrics ([Counters](https://micrometer.io/docs/concepts#_counters)):
+
+   ```
+      // Global counters
+      pravega_segmentstore.segment.read_bytes_global
+      pravega_segmentstore.segment.write_bytes_global
+      pravega_segmentstore.segment.write_events_global
+
+      // Per segment counters - all with tags {"scope", $scope, "stream", $stream, "segment", $segment, "epoch", $epoch}
+
+      pravega_segmentstore.segment.write_bytes
+      pravega_segmentstore.segment.read_bytes
+      pravega_segmentstore.segment.write_events
+  ```
+
+- Segment Store cache Read/Write latency Metrics ([Histogram](https://micrometer.io/docs/concepts#_histograms_and_percentiles)):
+
+  ```
+    pravega_segmentstore.cache.insert_latency_ms
+    pravega_segmentstore.cache.get_latency
+  ```
+
+- Segment Store cache Read/Write Metrics ([Counters](https://micrometer.io/docs/concepts#_counters)):
+
+  ```
+    pravega_segmentstore.cache.write_bytes
+    pravega_segmentstore.cache.read_bytes
+  ```
+
+- Segment Store cache size ([Gauge](https://micrometer.io/docs/concepts#_gauges)) and generation spread ([Histogram](https://micrometer.io/docs/concepts#_histograms_and_percentiles)) Metrics:
+
+  ```
+    pravega_segmentstore.cache.size_bytes
+    pravega_segmentstore.cache.gen
+  ```
+
+- Tier 1 Storage `DurableDataLog` Read/Write latency and queuing Metrics ([Histogram](https://micrometer.io/docs/concepts#_histograms_and_percentiles)):
+
+  ```
+    pravega_segmentstore.bookkeeper.total_write_latency_ms
+    pravega_segmentstore.bookkeeper.write_latency_ms
+    pravega_segmentstore.bookkeeper.write_queue_size
+    pravega_segmentstore.bookkeeper.write_queue_fill
+  ```
+
+- Tier 1 Storage `DurableDataLog` Read/Write ([Counter](https://micrometer.io/docs/concepts#_counters)) and per-container ledger count Metrics ([Gauge](https://micrometer.io/docs/concepts#_gauges)):
+
+  ```
+    pravega_segmentstore.bookkeeper.write_bytes
+    pravega_segmentstore.bookkeeper.bookkeeper_ledger_count - with tag {"container", $containerId}
+
+  ```
+
+- Tier 2 Storage Read/Write latency Metrics ([Histogram](https://micrometer.io/docs/concepts#_histograms_and_percentiles)):
+
+  ```
+    pravega_segmentstore.storage.read_latency_ms
+    pravega_segmentstore.storage.write_latency_ms
+  ```
+
+- Tier 2 Storage Read/Write data and file creation Metrics ([Counters](https://micrometer.io/docs/concepts#_counters)):
+
+  ```
+    pravega_segmentstore.storage.read_bytes
+    pravega_segmentstore.storage.write_bytes
+    pravega_segmentstore.storage.create_count
+  ```
+
+- Segment Store container-specific operation Metrics:
+
+  ```
+    // Histograms - all with tags {"container", $containerId}
+
+    segmentstore.container.process_operations.latency_ms
```

Comment:
One more reason to omit prefix to make all names consistent.

Reply:
ok